src/askscript/__tests__/09-operators/operators-07-lessThanOrEqual-oper.test.ts

```diff
@@ -0,0 +1,11 @@
+import { resources } from '../../../askvm';
+import { e2e } from '../../../utils/tools';
+
+test('lessThanOrEqual', async () => {
+  const environment = { resources };
+  expect(await e2e('4 <= 5', environment)).toBe(true);
```

Comment:
Side note: this is one of a few ways to test asynchronous code with Jest: https://jestjs.io/docs/en/asynchronous